charts/intel-resource-drivers/templates/_helpers.tpl

```diff
@@ -0,0 +1,79 @@
+{{/* Define common helpers for both GPU and Gaudi */}}
+{{- define "resource-drivers.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
+{{- end }}
+
+{{/* GPU specific helpers */}}
+{{- define "intel-gpu-resource-driver.name" -}}
+{{- default .Chart.Name .Values.gpu.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{- define "intel-gpu-resource-driver.fullname" -}}
+{{- printf "%s-gpu" (include "intel-gpu-resource-driver.name" .) | trunc 63 | trimSuffix "-" -}}
+{{- end }}
+
+{{- define "intel-gpu-resource-driver.namespace" -}}
+{{- default .Release.Namespace .Values.gpu.namespaceOverride }}
+{{- end }}
+
+{{- define "intel-gpu-resource-driver.labels" -}}
+app.kubernetes.io/name: {{ include "intel-gpu-resource-driver.fullname" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+helm.sh/chart: {{ include "resource-drivers.chart" . }}
+{{- end }}
+
+{{- define "intel-gpu-resource-driver.templateLabels" -}}
+app.kubernetes.io/name: {{ include "intel-gpu-resource-driver.name" . }}
+{{- end }}
+
+{{- define "intel-gpu-resource-driver.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "intel-gpu-resource-driver.name" . }}
+{{- end }}
+
+{{- define "intel-gpu-resource-driver.serviceAccountName" -}}
+{{- if .Values.gpu.serviceAccount.create -}}
+{{- default "intel-gpu-sa" .Values.gpu.serviceAccount.name -}}
+{{- end -}}
+{{- end }}
+
+{{/* Define full image name for GPU */}}
+{{- define "intel-gpu-resource-driver.fullimage" -}}
+{{- printf "%s/%s:%s" .Values.gpu.image.repository .Values.gpu.image.name .Values.gpu.image.tag -}}
```

Comment:
Gaudi one has `latest` as default, this does not?

Reply:
fixed